### PR TITLE
reverted EmptySearchResult, copied newest develop branch, formatted S…

### PR DESCRIPTION
**Pull request by lifeeeDEV**
# Description

Reverted other documents such as EmptySearchResult. Formatted LoadingIndicator into LF. no lint problems checked with `npm run lint`. Removed src/__tests__/screens/SearchResultsScreen.test.tsx -> Test for `Load More Button`

## Checklist

- [x] Screenshots added for any UX changes
- [x] Demos attached for animations/interactions
- [x] Pointed at proper branch (`develop` not `main`)
- [x] Assigned PR to yourself
- [ ] Requested review from code owners
- [ ] Relevant labels added (`feature`, `documentation`, etc.)
- [x] `Closes #<issue-number>` added if this resolves a GitHub issue
- [x] Branch is up to date with develop, ran `git rebase develop` if necessary
- [x] All GitHub Actions are passing
- [ ] `npm run todo-ci` ran if any todo comments were created

### CODE REVIEW

File: .env.template
Comment (Thenlie) [inline .env.template:1]: Why was this deleted?

File: src/__tests__/screens/SearchResultsScreen.test.tsx [hunk 1]
```diff
@@ -78,22 +80,5 @@ describe('Search Results Screen', () => {
...
-            loading: false,
-            moreToFetch: true,
-            refetch: () => {},
-        });
-
-        render(<RouterProvider router={router} />);
-
-        await screen.findByTestId('search-results-screen');
-        expect(screen.getByText('Load More')).toBeInTheDocument();
-        expect(screen.getByText('Load More')).not.toHaveAttribute('disabled');
```
Comment (Thenlie) [inline src/__tests__/screens/SearchResultsScreen.test.tsx:97]: Ideally we could test for the loading spinner, but I think I am fine to break that out into a new issue as it might be difficult.

**Review by Thenlie — CHANGES_REQUESTED** [review]
The loading indicator itself looks good! Just a couple questions about the deleted file and test.

File: .env.template
Comment (lifeeeDEV) [reply .env.template:1]: It was by accident, I am sorry.

File: src/__tests__/screens/SearchResultsScreen.test.tsx (hunk 1 above)
Comment (lifeeeDEV) [reply src/__tests__/screens/SearchResultsScreen.test.tsx:97]: I removed the tests for the Load More Button since it was replaced with the LoafingIndicator

File: .env.example [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Create .env file and add these environment variables.
-# Use `npm run db-spart-env` to auto-generate the supabase keys.
```
Comment (Thenlie) [inline .env.example:2]: Nice catch XD

**Review by Thenlie — APPROVED** [review]
Awesome! Thanks for the contribution and welcome to the Stream Team! 🥳
